### ISSUE TITLE
"no matching creation rules found" for helm-secrets version 2.0.0

### ISSUE THREAD

**Issue by max-rocket-internet** [futuresimple/helm-secrets#92]
I run `helm plugin update secrets` and got version `2.0.0` earlier today.

This newer version produces this error where version `1.3.1` worked fine:

```
$ helm secrets enc kubernetes/charts/apps/xxx/values/xxx/secrets.yaml
Encrypting kubernetes/charts/apps/xxx/values/xxx/secrets.yaml
error loading config: no matching creation rules found
Error: plugin "secrets" exited with error
```

I noticed there is no `2.0.0` release on Github. What's the deal?

**Comment by dkoleary88** [comment]
Same issue here. Had to downgrade to v1.3.1.

**Comment by max-rocket-internet** [comment]
The fix:

```
helm plugin remove secrets
helm plugin install --version 1.3.1 https://github.com/futuresimple/helm-secrets
```

**Comment by jonathancolby-olx** [comment]
same here with version `2.0.1`.

dir structure:

```
.
├── .sops.yaml
├── mysubdirectory
│   ├── secrets.yaml
│   ├── secrets.yaml.dec
```

.sops.yaml

```
---
creation_rules:
- path_regex: mysubdirectory/secrets.yaml
  kms: 'arn:aws:kms:eu-west-1:my-kms-key-redacted'
```

`helm secrets dec  mysubdirectory/secrets.yaml`   (works OK)

```
helm secrets dec  mysubdirectory/secrets.yaml
Encrypting mysubdirectory/secrets.yaml
error loading config: no matching creation rules found
Error: plugin "secrets" exited with error```
```

and original `mysubdirectory/secrets.yaml` is destroyed (emptied)

**Comment by mateuszzawisza** [comment]
Just merged #117  which seems to solve the issue. Please let me know if you experience any problems if using master.

**Comment by mhyllander** [comment]
@mateuszzawisza I think your change may have caused a regression. See my reasoning in https://github.com/futuresimple/helm-secrets/issues/121#issuecomment-505027232.

**Comment by mhyllander** [comment]
There are some issues with sops `path_regex`:

mozilla/sops#465
mozilla/sops#480

The first issue was fixed in the latest sops 3.3.1. Using that sops version the creation rules above work for me without #117.
